frontend/pages/admin/index.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-tabs left color="cyan" dark icons-and-text>
+    <v-tabs-slider color="yellow"></v-tabs-slider>
```

Comment:
You don't need to create a variable `tabs` that contains the data in order to use `v-tab`.
You can simply put the data directly in the html, the data are hardcoded anyway and there is only 2 elements.
The simpler is the better even if there is a little bit of duplication.
Example:
```suggestion
    <v-tabs-slider color="yellow"></v-tabs-slider>
      <v-tab href="#allowances">
        Allowances <v-icon>attach_money</v-icon>
      </v-tab>
      <v-tab href="#distributionConfigs">
        Distribution Configs <v-icon> blur_on </v-icon>
      </v-tab>

     <v-tab-item value="allowances">
      <v-card flat>
        <v-card-text>
          <Allowance />
        </v-card-text>
      </v-card>
    </v-tab-item>

     <v-tab-item value="distributionConfigs">
      <v-card flat>
        <v-card-text>
          <DistributionConfigure />
        </v-card-text>
      </v-card>
    </v-tab-item>

  </v-tabs>
    
    
```

Reply:
Done, thanks.